src/sentry/integrations/utils/code_mapping.py

```diff
@@ -109,7 +109,6 @@ def __init__(self, frame_file_path: str) -> None:
             or frame_file_path[0] in ["[", "<"]
             or frame_file_path.find(" ") > -1
             or frame_file_path.find("\\") > -1  # Windows support
-            or frame_file_path.find("/") == -1
```

Comment:
This is what prevented the use of files at the top-level for a source repository.